<?xml version="1.0" encoding="UTF-8"?>
<launch>

   <node name="robotnik_leds_driver" pkg="robotnik_leds" type="robotnik_leds_driver" respawn="false" output="screen">
      <param name="service_name" value="server_led_com" />
      <param name="path_device" value="/dev/ttyUSB_LEDS" />
   </node>

   <node name="robotnik_leds_control" pkg="robotnik_leds" type="robotnik_leds_control" respawn="false" output="screen">
      <param name="service_name" value="robotnik_leds_driver/server_led_com" />
      
      <param name="topic_direction" value="/rb2/robotnik_base_control/cmd_vel" />
      <param name="topic_elevator" value="/rb2/robotnik_base_control/elevator_status" />
      <param name="topic_estop" value="/rb2/robotnik_base_hw/emergency_stop" />
      
      <param name="max_value_lineal_x" value="1.0" />
      <param name="max_value_lineal_y" value="1.0" />
      <param name="max_value_angular_z" value="1.0" />
      <param name="porc_min_linear_x" value="0.0" />
      <param name="porc_min_linear_y" value="0.0" />
      <param name="porc_min_angular_z" value="0.0" />
      
      <param name="leds" value="134" />
      <param name="led_start_front" value="125" />
      <param name="led_end_front" value="7" />
      <param name="led_start_right" value="89" />
      <param name="led_end_right" value="129" />
      <param name="led_start_back" value="62" />
      <param name="led_end_back" value="79" />
      <param name="led_start_left" value="17" />
      <param name="led_end_left" value="57" />
      <param name="led_start_front_right" value="114" />
      <param name="led_end_front_right" value="125" />
      <param name="led_start_front_left" value="7" />
      <param name="led_end_front_left" value="17" />
      <param name="led_start_back_left" value="52" />
      <param name="led_end_back_left" value="62" />
      <param name="led_start_back_right" value="79" />
      <param name="led_end_back_right" value="89" />
   </node>
</launch>
